### PR TITLE
Decode html entities

### PR DESCRIPTION
**Pull request by andrispraulitis**
#24

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -22,8 +22,12 @@ var Script2 = {
         if (!this.src) {
           Script2.p = Script2.p.then(() => {
             var s = document.createElement('script')
+            var h = _this.$el.innerHTML;
```
Comment (taoeffect) [inline src/index.js]: `_this` doesn't exist.

Comment (andrispraulitis) [reply src/index.js]: sorry

File: src/index.js [hunk 2]
```diff
@@ -22,8 +22,12 @@ var Script2 = {
         if (!this.src) {
           Script2.p = Script2.p.then(() => {
             var s = document.createElement('script')
+            var h = this.$el.innerHTML;
+            h = h.replace(/&lt;/gi, '<');
+            h = h.replace(/&gt;/gi, '>');
+            h = h.replace(/&amp;/gi, '&');
```
Comment (taoeffect) [inline src/index.js]: These `h = h`'s can be shortened to `var h = this.$el.innerHTML.replace( .. ).replace( .. ). replace( .. )`.

Also, don't forget to remove semi-colons to ensure [`standard`](https://github.com/standard/standard) linter passes.

**Comment by taoeffect** [comment]
Ok, this looks good, thank you!

**Comment by andrispraulitis** [comment]
@taoeffect I still had other problems with spacing and having js variables withing image tag src but was able to fix them my side. Just had a rush project and these entities fixes where the only ones I could not fix my side.

**Comment by taoeffect** [comment]
Thanks for the PR! 2.0.3 has been pushed out with these fixes. :)

**Pull request merged**